soh/soh/Enhancements/randomizer/context.h

```diff
@@ -123,6 +123,11 @@ class Context {
     void SetEventChkInf(int32_t flag, bool state);
     uint8_t GetAmmo(uint32_t item);
     void SetAmmo(uint32_t item, uint8_t count);
+    void StartPerformanceTimer(PerformanceTimers timer);
```

Comment:
Nice, this is mostly what I meant, but additionally I think it should just be split off into its own module, maybe in developer tools. (Eg if we wanted to benchmark something outside of the context of a randomizer)